config/secrets.yml

```diff
@@ -17,28 +17,9 @@
 
 # Environmental secrets are only available for that specific environment.
 
-default: &base
-  support_provider_key: <%= ENV['SUPPORT_PROVIDER_KEY'] %>
-  support_provider_key_preview: <%= ENV['SUPPORT_PROVIDER_KEY_PREVIEW'] %>
-
-development:
-  <<: *base
-
-test:
-  <<: *base
-
-# Do not keep preview secrets in the unencrypted secrets file.
-# Instead, either read values from the environment.
-# Or, use `bin/rails secrets:setup` to configure encrypted secrets
-# and move the `preview:` environment over there.
-preview:
-  <<: *base
-  secret_key_base: <%= ENV['SECRET_KEY_BASE'] %>
-
 # Do not keep production secrets in the unencrypted secrets file.
 # Instead, either read values from the environment.
 # Or, use `bin/rails secrets:setup` to configure encrypted secrets
 # and move the `production:` environment over there.
 production:
-  <<: *base
   secret_key_base: <%= ENV['SECRET_KEY_BASE'] %>
```

Comment:
This line sohuld also not be needed. Maybe remove the whole file?

Reply:
Sounds good :+1: